src/App.tsx

```diff
@@ -175,6 +175,7 @@ const AppRoutes = ({ base }: AppProps) => {
                 <Route path="404" element={<NotFound />} />
                 <Route path="403" element={<AccessDenied />} />
                 <Route path="*" element={<NotFound />} />
+                <Route path="slug/:articleId" element={<PlainArticlePage />} />
```

Comment:
Skal nok ikkje ha slug i path nei. I verste fall /p/ og i beste fall ingenting. Altså `:articleId`

Reply:
Kan ta og sjekke det! Bare la inn slug midlertid for at noen skulle kommentere